client-api/src/main/kotlin/org/ostelco/prime/client/api/TopupModule.kt

```diff
@@ -37,27 +39,33 @@ class TopupModule : PrimeModule {
 
     private val storage by lazy { getResource<Storage>() }
     private val ocsSubscriberService by lazy { getResource<OcsSubscriberService>() }
+    private val LOG by logger()
 
     override fun init(env: Environment) {
 
         val dao = SubscriberDAOImpl(storage, ocsSubscriberService)
         val jerseyEnv = env.jersey()
 
+        val client: Client = JerseyClientBuilder(env)
+                .using(config.jerseyClientConfiguration)
+                .using(ObjectMapper()
+                        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false))
+                .build(env.getName())
+        client.property(ClientProperties.CONNECT_TIMEOUT, 2000)
+        client.property(ClientProperties.READ_TIMEOUT, 2000)
+
         /* APIs. */
         jerseyEnv.register(AnalyticsResource(dao))
         jerseyEnv.register(ConsentsResource(dao))
         jerseyEnv.register(ProductsResource(dao))
         jerseyEnv.register(ProfileResource(dao))
-        jerseyEnv.register(SubscriptionResource(dao))
+
+        LOG.info("client-api pseudonymEndpoint ${config.pseudonymEndpoint}")
```

Comment:
No need for log if `pseudonymEndpoint` is set in `prime/config/config.yaml`.

Reply:
Done